mbuild/packing.py

```diff
@@ -640,7 +662,17 @@ def _packmol_error(out, err):
 
 
 def _run_packmol(input_text, filled_xyz, temp_file):
+    """Apply PACKMOL packing based on the input text.
 
+    Parameters
```

Comment:
If they're all required, do we need to say so? I usually omit if not required. Not sure which is formally correct

Reply:
Makes sense, and that follows the numpydoc codestyle. Will update.